### PR TITLE
Add basic tests and Travis implementation

### PR DESCRIPTION
**Pull request by chrisminett**
Includes adding support for php70

### CODE REVIEW

**Comment by chrisminett** [comment]
@Pudge601

File: phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="vendor/autoload.php" colors="true">
+    <testsuites>
+        <testsuite name="Mxm">
+            <directory>tests</directory>
```
Comment (Pudge601) [inline phpunit.xml.dist]: Isn't this `tests/unit`? I know it probably doesn't make a difference, but that's the directory with the unit tests in.

Comment (chrisminett) [inline phpunit.xml.dist]: Good spot. Hadn't noticed, as it worked.

File: composer.json [hunk 2]
```diff
@@ -25,5 +25,6 @@
     },
```
Comment (Pudge601) [inline composer.json]: I think the "autoload-dev" above needs changing to
```
        "psr-4": {
            "Mxm\\Test\\": "tests/unit/"
        }
```

**Comment by Pudge601** [comment]
All looks good to me, shall I merge?

**Comment by chrisminett** [comment]
Hoping that @qubit05 will take a quick look too.

**Comment by qubit05** [comment]
Yer looks fine.

**Pull request merged**